### PR TITLE
[Google Blockly] Implement trashcan

### PR DESCRIPTION
**Pull request by ajpal**
This sort of splits the difference between our existing trashcan and Google's default one.
- It's the same trashcan shape as Google Blockly's, though we change the color to Code.org purple
- It's positioned over the toolbox instead of in the bottom corner
- I added the 🚫  signifier for when you can't delete the block. Google's UI hint is just that the lid doesn't open, but I figure we might want our more explicit UI here.
- The trashcan shows any time you're dragging a block. I haven't implemented the fade in UI that we used to have. I think this is okay for now, and can be added later if needed.

Sep-30-2020 15-53-51

### CODE REVIEW

File: apps/src/blocklyAddons/cdoTrashcan.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import GoogleBlockly from 'blockly/core';
+
+export default class CdoTrashcan extends GoogleBlockly.Trashcan {
+  /** Use our trash png and add ⃠ for undeletable blocks
```
Comment (breville) [inline apps/src/blocklyAddons/cdoTrashcan.js]: nit: This character just shows as a box?

File: apps/src/blocklyAddons/cdoTrashcan.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    Blockly.utils.dom.createSvgElement(
+      'line',
+      {x1: 0, y1: 10, x2: 45, y2: 60, stroke: '#c00', 'stroke-width': 5},
+      this.notAllowed_
+    );
+    Blockly.utils.dom.createSvgElement(
+      'circle',
+      {cx: 22, cy: 33, r: 33, stroke: '#c00', 'stroke-width': 5, fill: 'none'},
+      this.notAllowed_
+    );
```
Comment (breville) [inline apps/src/blocklyAddons/cdoTrashcan.js:36]: Are these the circle with the line going through it?

**Review by breville — APPROVED** [review]
Very fun!

File: apps/src/blocklyAddons/cdoTrashcan.js (hunk 1 above)
Comment (ajpal) [reply apps/src/blocklyAddons/cdoTrashcan.js]: whoops, it shows up in my editor, but changed to "circle with line through it" (side note: I really wish that symbol had a better name)

File: apps/src/blocklyAddons/cdoTrashcan.js (hunk 2 above)
Comment (ajpal) [reply apps/src/blocklyAddons/cdoTrashcan.js:36]: Yep, looks like this:
Oct-05-2020 13-00-26

File: apps/src/blocklyAddons/cdoTrashcan.js (hunk 1 above)
Comment (breville) [reply apps/src/blocklyAddons/cdoTrashcan.js]: Some potential names at https://en.wikipedia.org/wiki/No_symbol :)

**Pull request merged**